src/transformers/modeling_electra.py

```diff
@@ -570,10 +570,9 @@ def forward(
         position_ids=None,
         head_mask=None,
         inputs_embeds=None,
-        masked_lm_labels=None,
```

Comment:
I think `masked_lm_labels` was deleted accidently here

Reply:
Renamed to `labels`